core/server/worker/src/main/java/alluxio/worker/page/PagedBlockStore.java

```diff
@@ -67,6 +67,7 @@ public class PagedBlockStore implements BlockStore {
   private final UfsInputStreamCache mUfsInStreamCache = new UfsInputStreamCache();
   private final List<BlockStoreEventListener> mBlockStoreEventListeners =
       new CopyOnWriteArrayList<>();
+  private final List<PageStoreDir> mPageStoreDirs;
```

Comment:
I put list of page storage dirs in `PageMetaStore` and added a method on the interface. See #15903.

Reply:
Cool, I will update this pr after your PR merged, thanks!